scripts/camViewer

```diff
@@ -320,17 +315,18 @@ name_to_pv "$CAMNAME" "$hutch"
 if [ $MAINSCREEN -gt 0 ]; then
     tmp=/tmp/cv.$$
     list_cams "$hutch" >$tmp
-    c=$(wc <$tmp | grep "$CAMNAME" $tmp | awk '{print $1}')
+
+    c=$(wc $tmp | grep "$CAMNAME" $tmp | awk '{print $1}') # not sure what is happening here
```

Comment:
This wc is also a mystery to me, I think it must be an error. This will always land in the "camera name not found in hutch" branch, which I've noticed when using the script!

Reply:
So, revisiting here, probably we should be grepping through the config file instead of word-counting the temp file? This is definitely a pre-existing bug though and you're not obligated to mix this sort of thing with a style PR

Reply:
There is no 'the' config file - the config file can have includes and it's not clear to me that we want to write parsing for this.

Reply:
Are you recommending removing this check altogether?

Reply:
Looking more closely: the `list_cams` function here already does a half-trying version of working through the includes (hard-coded).
So, the quick fix is to the simply remove the wc clause and start from the grep, but whether or not this is a thing we should do at all is another question.